Makefile

```
@@ -60,6 +61,10 @@ uninstall:
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/python-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@jupyterlab/toc)
 	pip uninstall -y jupyterlab-git
+	pip uninstall -y jupyter-lsp
+	jupyter labextension uninstall @krassowski/jupyterlab-lsp
```

These lines uninstalls old versions of the extension. 
It is recommended as noted in https://github.com/krassowski/jupyterlab-lsp/releases/tag/v3.0.0

---

We need to ignore the exit code in case `jupyter` is not installed

```suggestion
	pip uninstall -y jupyter-lsp
	- jupyter labextension uninstall @krassowski/jupyterlab-lsp
```